app/src/main/res/layout/topic_overview_fragment.xml

```diff
@@ -2,17 +2,121 @@
+        android:textColor="@color/oppiaPrimaryText"
+        android:textSize="16sp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/topic_name_text_view"/>
+      <TextView
+        android:id="@+id/topic_description_text_view"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="24dp"
+        android:layout_marginTop="16dp"
+        android:layout_marginEnd="24dp"
+        android:gravity="start"
+        android:text="@{viewModel.topic.description}"
+        android:textColor="@color/oppiaPrimaryText"
+        android:textSize="16sp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/story_count_text_view"/>
+      <TextView
+        android:id="@+id/see_more_text_view"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="24dp"
+        android:layout_marginTop="16dp"
+        android:layout_marginEnd="24dp"
+        android:gravity="start"
+        android:onClick="@{presenter::seeMoreClicked}"
+        android:text="@string/topic_overview_see_more"
+        android:textColor="@color/colorPrimary"
+        android:textSize="16sp"
+        android:textStyle="bold"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/topic_description_text_view"/>
+      <ImageView
+        android:id="@+id/download_status_image_view"
+        android:src="@{viewModel.downloadStatus, default=@drawable/ic_file_download_primary_24dp}"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="24dp"
+        android:layout_marginTop="24dp"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/see_more_text_view"/>
+      <TextView
+        android:id="@+id/download_story_count_text_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
+        android:text="@{viewModel.topic.storyCount + ` Stories`}"
+        android:textColor="@color/oppiaPrimaryText"
+        android:textSize="16sp"
+        app:layout_constraintBottom_toBottomOf="@+id/download_status_image_view"
+        app:layout_constraintStart_toEndOf="@+id/download_status_image_view"
+        app:layout_constraintTop_toTopOf="@+id/download_status_image_view"/>
+      <TextView
+        android:id="@+id/download_size_text_view"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="8dp"
+        android:text="@{`(` + viewModel.topic.getSerializedSize() + ` MB)`}"
```

Comment:
Ditto here (but instead of plurals, we should just use a string with variables rather than using concatenation to ensure RTL cases are properly handled).

Reply:
Resolved as above.

Reply:
This appears to still be using string concatenation. Prefer using a string with variables, instead, to ensure RTL cases are handled for this text view.